engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -69,19 +82,33 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId, override val ioActor: Act
   }
   
   private def getOutputFilePaths(workflowOutputsPath: Path): List[(Path, Path)] = {
+
+    val flattenWorkflowOutputs: Boolean = workflowDescriptor.getWorkflowOption(FlattenWorkflowOutputs) match {
```

Comment:
This `match` is really just `.exists("true")`

Reply:
`contains("true")`

Reply:
🤦‍♂️! ... thanks @mcovarr